modules/ROOT/nav.adoc

```diff
@@ -385,6 +385,10 @@
 ** xref:tinymce-and-cors.adoc[Cross-Origin Resource Sharing (CORS)]
 * Release information
 ** xref:release-notes.adoc[Release notes for TinyMCE 6]
+*** TinyMCE 6.1.2
```

Comment:
We also need to add this entry to the `release-notes.adoc` index page.

Reply:
Thanks for the heads-up: added and pushed to the branch. Also added this to my general To-Do doc regarding release notes.